packages/apollo-engine-reporting/src/agent.ts

```diff
@@ -239,6 +250,25 @@ export interface EngineReportingOptions<TContext> {
    */
   generateClientInfo?: GenerateClientInfo<TContext>;
 
+  /**
+   * Enable experimental apollo schema reporting, letting the server report its schema to the Apollo registry.
+   * This will start a thread within the apollo agent that will periodically report server info and a schema potentially
+   * Look at https://github.com/apollographql/apollo-schema-reporting-preview-docs for more information about configuration and use cases.
```

Comment:
Let's not put the preview docs in the source code; I fear we'll forget to remove it when this is out of preview.

Reply:
We have to change this from experimental to not experimental so I'd prefer to leave it here for now.